### DISCUSSION TITLE
Docs and maps

### DISCUSSION THREAD

**Discussion by IceflowRE** [redeclipse/base#1122] [Help and Support] [answered]
I already added two maps to the docs (https://github.com/red-eclipse/docs/tree/master/maps).
- Is this location ok?
  - if the location is ok, the toc generation has to be added here.
- How should the file names named? (Just like the map itself or `Map-<map name>`.)

The information about the maps, is it ok like it is or some suggestions?

**Comment by qreeves** [comment, marked as answer]
You likely don't even need the `Map-` prefix if you're using a sub-directory, but I will need to add support for it to the website builds.
